src/client/scripts.js

```diff
@@ -9,15 +9,23 @@ require('magnific-popup/dist/magnific-popup.css');
 
 // Local css
 require(sourceDirectoryForCssDependencies + 'index.css');
+require(sourceDirectoryForCssDependencies + 'cover.css');
 
 // Local js dependencies
 
 require(sourceDirectoryForJsDependencies + 'modules/Invitation.js');
 require(sourceDirectoryForJsDependencies + 'models/Invitation.js');
 require(sourceDirectoryForJsDependencies + 'modules/connection.js');
 require(sourceDirectoryForJsDependencies + 'modules/endpointBases.js');
+require(sourceDirectoryForJsDependencies + 'modules/LoaderSpinner.js');
 require(sourceDirectoryForJsDependencies + 'modules/markupControllers/confirmation.js');
 require(sourceDirectoryForJsDependencies + 'mainAppLoader.js');
 require(sourceDirectoryForJsDependencies + 'theme/theme.js');
+
+$(document).ready(function() {
```

Comment:
Can you please use this loader in here? https://github.com/vctr90/wedding-app/blob/development/src/client/app/mainAppLoader.js#L31